src/enums.h

```diff
@@ -637,4 +637,24 @@ struct Reflect {
 	uint16_t chance = 0;
 };
 
+enum ClientElement {
```

Comment:
```suggestion
enum ClientDamageType {
```
DamageType looks good

![image](https://user-images.githubusercontent.com/53269692/159610542-8d08e979-fa31-4a8f-b429-be032b5f35c4.png)

Reply:
`CLIENT_DAMAGETYPE`?